07/22.py

```diff
@@ -1,8 +1,10 @@
 def list_all_keys_in_dict(my_dict):
     """ print all keys in the dictionary """
-    pass
-
-
+    list = []
```

Comment:
you should never use the name "list" as a variable name. it overrides the built-in keyword "list".